### PR TITLE
Prepare module for release, add initial version of 0.1.0

### PR DESCRIPTION
**Pull request by crhino**

### CODE REVIEW

File: test/hcp/testdata/vm.golden [hunk 1]
```diff
@@ -114,7 +116,8 @@ resource "hcp_consul_cluster_root_token" "token" {
 }
 
 module "vm_client" {
-  source = "../../modules/hcp-vm-client"
+  source  = "hashicorp/hcp-consul/azurerm//modules/hcp-vm-client"
```
Comment (chapmanc) [inline test/hcp/testdata/vm.golden:119]: Double slash?

File: hcp-ui-templates/vm/main.tf [hunk 2]
```diff
@@ -114,7 +116,8 @@ resource "hcp_consul_cluster_root_token" "token" {
 }
 
 module "vm_client" {
-  source = "../../modules/hcp-vm-client"
+  source  = "hashicorp/hcp-consul/azurerm//modules/hcp-vm-client"
```
Comment (chapmanc) [inline hcp-ui-templates/vm/main.tf:119]: Double slash?

**Review by chapmanc — APPROVED** [review]
Minor double slash but the rest looks good!

File: test/hcp/testdata/vm.golden (hunk 1 above)
Comment (crhino) [reply test/hcp/testdata/vm.golden:119]: That is actually required: https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories

**Pull request merged**